src/test/java/homework17.java

```diff
@@ -22,23 +21,37 @@ public void addSongToPlaylist() {
 //String namePlaylist = UUID.randomUUID().toString();
 
 
-        WebElement search = driver.findElement(By.cssSelector("input[type='search']"));
-        search.sendKeys("F");
- WebElement brand = driver.findElement(By.cssSelector("[class='brand']"));
- brand.click();
-
-WebElement allViewButton = driver.findElement(By.cssSelector("[data-test='view-all-songs-btn']"));
-allViewButton.click();
-WebElement clickSong = driver.findElement(By.cssSelector("#songResultsWrapper > div > div > div.item-container > table > tr"));
-clickSong.click();
-WebElement addToButton = driver.findElement(By.cssSelector("[class='btn-add-to']"));
-addToButton.click();
-WebElement playlistNameNew = driver.findElement(By.xpath("//*[@id='queueWrapper']/header/div[3]/div/section[2]/form/input"));
-playlistNameNew.sendKeys("HW17");
-WebElement submitButton = driver.findElement(By.xpath("//*[@id='songResultsWrapper']/header/div[3]/div/section[2]/form/button"));
-submitButton.click();
-
-
+//        WebElement search = driver.findElement(By.cssSelector("input[type='search']"));
+//        search.sendKeys("F");
+// WebElement brand = driver.findElement(By.cssSelector("[class='brand']"));
+// brand.click();
+//
+//WebElement allViewButton = driver.findElement(By.cssSelector("[data-test='view-all-songs-btn']"));
+//allViewButton.click();
+//WebElement clickSong = driver.findElement(By.cssSelector("#songResultsWrapper > div > div > div.item-container > table > tr"));
+//clickSong.click();
+//WebElement addToButton = driver.findElement(By.cssSelector("[class='btn-add-to']"));
+//addToButton.click();
+//WebElement playlistNameNew = driver.findElement(By.xpath("//*[@id='queueWrapper']/header/div[3]/div/section[2]/form/input"));
+//playlistNameNew.sendKeys("HW17");
+//WebElement submitButton = driver.findElement(By.xpath("//*[@id='songResultsWrapper']/header/div[3]/div/section[2]/form/button"));
+//submitButton.click();
+//
+
+        public void addSongToPlayListTest() throws InterruptedException {
+            String songName = "Riqui-Riqui";
+        login("dmytro.kravchuk@testpro.io", "Fr440003");
+            WebElement searchInput = driver.findElement(By.cssSelector("#searchForm>input"));
+            searchInput.sendKeys(songName);
+            searchInput.sendKeys(Keys.ENTER);
+            Thread.sleep(3000);
+            WebElement likeButton = driver.findElement(By.xpath("//section[@id='searchExcerptsWrapper']//button[contains(@title, 'Riqui-Riqui')]/i[@data-test='btn-like-unliked']"));
+            likeButton.click();
+            WebElement favoritePlayListLink = driver.findElement(By.xpath("//li[contains(@class, 'favorites')]/a"));
+            favoritePlayListLink.click();
+            String favoriteSong = "//section[@id='favoritesWrapper']//td[contains(text(), '%s')]";
+            WebElement favoriteSongElement = driver.findElement(By.xpath(String.format(favoriteSong, songName)));
+            Assert.assertTrue(favoriteSongElement.isDisplayed());
```

Comment:
How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)